src/components/shared/MarkerTooltipContents.js

```diff
@@ -122,6 +122,20 @@ function _makePhaseTimesArray(
   return array;
 }
 
+function _dataPrioReplace(str: number) {
```

Comment:
Please use full words in function names, as opposed to abbreviations. This helps make the functions easier to understand. "Prio" is a bit a cryptic.

Reply:
The `str` argument isn't a string, so think this could be changed to a more accurate word.